analytics/src/analytics/integrations/etldb/migrations/versions/2024_11_15_update_columns_etldb.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE IF NOT EXISTS gh_schema_version (
```

Comment:
The schema_version table creation needs to go into it's own file

Also it should just be called `schema_version`, with no `gh_`

Reply:
done